flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobExceptionsInfo.java

```diff
@@ -100,6 +121,11 @@ public boolean isTruncated() {
         return truncated;
     }
 
+    @JsonIgnore
+    public List<JobExceptionHistoryEntry> getExceptionHistory() {
```

Comment:
Looking at it, I guess, we would have to introduce another class that holds the list and a `truncated` flag.